srcDLL/op2ext.cpp

```diff
@@ -108,3 +108,10 @@ OP2EXT_API bool IsIniModuleLoaded(const char* moduleName)
 {
 	return iniModuleLoader.IsModuleLoaded(moduleName);
 }
+
+OP2EXT_API size_t GetLoadedModuleCount()
+{
+	const std::size_t consoleModuleCount = (consoleModLoader.IsModuleLoaded() ? 1 : 0);
+
+	return iniModuleLoader.Count() + consoleModuleCount;
```

Comment:
Maybe the ternary operation should be moved into a method on `consoleModuleLoader` for consistency.

Reply:
The console module loader doesn't really make sense returning a count because it can only have 1 module. However, I like that this will make the ini module and console module loaders appear identical in function setup. I suppose it allows easier expansion if we ever support multiple console modules.

Reply:
Yes, good point, having the console module loader only support 1 module seems more like a current implementation detail. It could potentially be upgraded to support multiple modules at some point in the future.